packages/land/contracts/PolygonLand.sol

```diff
@@ -0,0 +1,205 @@
+import {IERC2981Upgradeable} from "@openzeppelin/contracts-upgradeable/interfaces/IERC2981Upgradeable.sol";
+import {IRoyaltyManager} from "@sandbox-smart-contracts/dependency-royalty-management/contracts/interfaces/IRoyaltyManager.sol";
+
+/// @title LAND token on L2
+contract PolygonLand is PolygonLandBaseToken, ERC2771Handler, OperatorFiltererUpgradeable {
+    using AddressUpgradeable for address;
+
+    uint16 internal constant TOTAL_BASIS_POINTS = 10000;
+
+    IRoyaltyManager private royaltyManager;
+    address private _owner;
+
+    event OperatorRegistrySet(address indexed registry);
+    event RoyaltyManagerSet(address indexed _royaltyManager);
+    event OwnerSet(address indexed owner);
+
+    function initialize(address trustedForwarder, address _royaltyManager, address owner) external initializer {
+        _admin = _msgSender();
+        __ERC2771Handler_initialize(trustedForwarder);
+        _setRoyaltyManager(_royaltyManager);
+        _setOwner(owner);
+        emit AdminChanged(address(0), _admin);
+    }
+
+    /// @notice set royalty manager
+    /// @param _royaltyManager address of royalty manager to set
+    function _setRoyaltyManager(address _royaltyManager) internal {
+        royaltyManager = IRoyaltyManager(_royaltyManager);
+        emit RoyaltyManagerSet(_royaltyManager);
+    }
+
+    function _setOwner(address newOwner) internal {
+        _owner = newOwner;
+        emit OwnerSet(newOwner);
+    }
+
+    /// @dev Change the address of the trusted forwarder for meta-TX
+    /// @param trustedForwarder The new trustedForwarder
+    function setTrustedForwarder(address trustedForwarder) external onlyAdmin {
+        _trustedForwarder = trustedForwarder;
+        emit TrustedForwarderSet(trustedForwarder);
+    }
+
+    /// @notice Returns how much royalty is owed and to whom based on ERC2981
+    /// @dev tokenId is one of the EIP2981 args for this function can't be removed
+    /// @param _salePrice the price of token on which the royalty is calculated
+    /// @return receiver the receiver of royalty
+    /// @return royaltyAmount the amount of royalty
+    function royaltyInfo(
+        uint256 /*_tokenId */,
+        uint256 _salePrice
+    ) external view returns (address receiver, uint256 royaltyAmount) {
+        uint16 royaltyBps;
+        (receiver, royaltyBps) = royaltyManager.getRoyaltyInfo();
+        royaltyAmount = (_salePrice * royaltyBps) / TOTAL_BASIS_POINTS;
+        return (receiver, royaltyAmount);
+    }
+
+    /// @notice returns the royalty manager
+    /// @return royaltyManagerAddress address of royalty manager contract.
+    function getRoyaltyManager() external view returns (IRoyaltyManager royaltyManagerAddress) {
+        return royaltyManager;
+    }
+
+    /// @notice returns the owner address
+    /// @return owner address of contract owner
+    function getOwner() external view returns (address owner) {
```

Comment:
need to follow the actual standard 
https://eips.ethereum.org/EIPS/eip-173#specification